lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -75,6 +95,36 @@ def _reserve_task(course_id, task_type, task_key, task_input, requester):
     return InstructorTask.create(course_id, task_type, task_key, task_input, requester)
 
 
+def generate_already_running_error_message(task_type):
+    """
+
+    Returns already running error message for given task type.
+    """
+
+    message = ''
+    already_running_message_template = "The {} report is being created. " \
```

Comment:
For translation, you'll want to change `{}` to something like `{report_type}` so translators know what string will get substituted there.